src/utils.js

```diff
@@ -85,6 +85,41 @@ exports.flattenStartCase = (object) => {
 };
 
 /**
+ * @summary Remove paths contained in strings
+ * @function
+ * @public
+ *
+ * @param {String} str - string potentially containing paths
+ * @returns {String} string with basenamed paths if any
+ *
+ * @example
+ */
+exports.removeContainedPaths = (str) => {
+  const absolutify = (pathname) => {
```

Comment:
You're right, I wanted to push out the changes without brushing them up because I hit the paths-with-spaces issue and wanted feedback on that. I'll make sure to change it.